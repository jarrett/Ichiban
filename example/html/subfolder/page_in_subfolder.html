<h1>A Page in a Subfolder</h1>