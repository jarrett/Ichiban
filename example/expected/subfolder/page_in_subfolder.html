<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <h1>A Page in a Subfolder</h1>
  </body>
</html>